admin_manual/configuration_user/user_auth_ldap.rst

```diff
@@ -384,32 +384,33 @@ Special Attributes
    :alt: Special Attributes.
 
 Quota Field:
-  ownCloud can read an LDAP attribute and set the user quota according to its
-  value. Specify the attribute here, and it will return human-readable values, 
-  e.g. "2 GB". Any quota set in LDAP overrides quotas set on the ownCloud user 
+  The name of the LDAP attribute to retrieve the user quota limit from, e.g., ``ownCloudQuota``. *Note:* any quota set in LDAP overrides quotas set in ownCloud's user 
   management page.
 
-  * Example: *ownCloudQuota*
-
 Quota Default:
-  Override ownCloud default quota for LDAP users who do not have a quota set in 
-  the Quota Field.
-
-  * Example: *15 GB*
+  Override ownCloud's default quota *for LDAP users* who do not have a quota set in 
+  the Quota Field, e.g., ``15 GB``.
 
 Email Field:
-  Set the user's email from their LDAP attribute. Leave it empty for default 
+  Set the user's email from an LDAP attribute, e.g., ``mail``. Leave it empty for default 
   behavior.
 
-  * Example: *mail*
-
 User Home Folder Naming Rule:
   By default, the ownCloud server creates the user directory in your ownCloud 
-  data directory and gives it the ownCloud username, .e.g ``/var/www/owncloud/data/alice``. You may want to override this setting and name it after an LDAP
-  attribute value. The attribute can also return an absolute path, e.g. 
+  data directory and gives it the ownCloud username, e.g., ``/var/www/owncloud/data/alice``. You may want to override this setting and name it after an LDAP
+  attribute value, e.g., ``cn``. The attribute can also return an absolute path, e.g. 
   ``/mnt/storage43/alice``. Leave it empty for default behavior.
 
-  * Example: *cn*
+Please bear in mind the following, when using these fields to assign user quota limits. 
+It should help to alleviate any, potential, confusion.
+
+1. After installation ownCloud uses a default quota value.
```

Comment:
Only when the administrator has set it, otherwise is **unlimited** by default 

I don't know how to put in words 😄 , an unlimited quota can be changed to a size defined quota.

In this case is important to mention that the LDAP/AD quota overrides oC quota/non quota.